aurget

```diff
@@ -333,6 +349,8 @@ set_defaults() {
   sync_mode='install'
   temp_directory='/tmp/aurget'
   user_config="${XDG_CONFIG_HOME:-$HOME/.config}/aurgetrc"
+  pkg_cache_dir=$(sed -n 's/^CacheDir[ \t]*\=[ \t]*//p' /etc/pacman.conf)
```

Comment:
Redirect error to /dev/null; default path defined later
pkg_cache_dir=$(sed -n 's/^CacheDir[ \t]*\=[ \t]*//p' /etc/pacman.conf 2>/dev/null)